crates/ruff_python_parser/tests/snapshots/invalid_syntax@async_unexpected_token.py.snap

```diff
@@ -217,6 +217,15 @@ Module(
   |
 
 
+  |
+5 | # TODO(dhruvmanila): Here, `match` is actually a Name token because
+6 | # of the soft keyword # transformer
+7 | async match test:
+  |             ^^^^ Syntax Error: Simple statements must be separated by newlines or semicolons
```

Comment:
Future: We should change the range to the start of the current token (an empty range), once our message rendering supports rendering empty ranges.